.ci_support/linux_64_cross_target_impl_platformlinux-64.yaml

```diff
@@ -8,6 +8,12 @@ channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
+cross_target_impl_platform:
+- linux-64
+cross_target_platform:
+- linux-64
+- linux-aarch64
+- linux-ppc64le
```

Comment:
However in some cases, `conda-smithy` is not building out all the `cross_target_impl_platform` first and instead building out all `cross_target_platform`, which would cause issues as packages would be missing

Reply:
I wasn't aware that conda-smithy had any concept of the dependency tree. You're saying conda-smithy knows what builds first and sets up the CI jobs accordingly? Even then, can you use `pin_subpackage` across independent CI jobs? I don't think we can express this with `conda_build_config.yaml`, which in my understanding only knows how to do outer products of the listed keys.

Reply:
Basically conda-smithy calls conda-build to render a recipe (playing around with the `conda render` CLI may help get a feel for this)

It's possible to express cartesian products in `conda_build_config.yaml`. This is how [variants work typically]( https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#aggregation-of-multiple-variants ). This behavior can be [altered in various ways]( https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#special-variant-keys )

Side note: Had also explored a Jinja `for`-loop as another approach, but that ran into a conda-build bug ( https://github.com/conda/conda-build/issues/2815 ). So didn't pursue that further